Update host monitoring dashboard, docs, config example (#585)
Dynatrace/dynatrace-otel-collector: docs/dashboards/Host Dashboard.md

### Adding attributes to the allow list

The following attributes are not included in the default allow list of resource attributes in Dynatrace:

- `host.arch`
- `host.ip`
- `host.name`
- `os.type`
- `process.command_line`
- `process.name`
- `process.pid`
    - CAUTION: This resource attribute only needs to be added when you have processes that spawn sub-processes to be

>       able to accurately show CPU/memory usage per process. Only add the attribute if you have a need for it, since it
>       is otherwise discouraged to use the `process.pid` as a resource attribute due to the risk of cardinality explosion
>       in your metrics.

Reviewer: ```suggestion
      able to accurately show CPU/memory usage per process. Only add the `process.pid` attribute if you have a need for it. Adding the attribute can lead to a cardinality explosion in your metrics.
```
